charts/kubearchive/templates/database/ps-pv.yaml

```diff
@@ -5,13 +5,13 @@ kind: PersistentVolume
 metadata:
   labels:
     type: local
-    app: {{ .Values.database.name | default "postgresql" }}
-  name: {{ .Values.database.name | default "postgresql" }}-volume
-  namespace: {{ .Values.kubearchive.namespace }}
+    app: {{ tpl .Values.database.name . }}
+  name: {{ tpl .Values.database.name . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I suggest removing this based on the fact that the `kubearchive` namespace is provided by the `-n` option in `helm install` command.

```suggestion
```

Reply:
removed the `namespace`